lib/ember-test-helpers/test-module.js

```diff
@@ -45,11 +45,6 @@ export default Klass.extend({
     this.setupSteps.push(this.setupContainer);
     this.setupSteps.push(this.setupContext);
     this.setupSteps.push(this.setupTestElements);
-
-    if (this.callbacks.setup) {
-      this.setupSteps.push( this.callbacks.setup );
-      delete this.callbacks.setup;
```

Comment:
Maybe to be clear we should introduce both `contextualizedSetupSteps` and `contextualizedTeardownSteps`, both of which will be invoked with the module's `context`.